src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Rename/RenamerProjectTreeActionHandler.HandleFileMove.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Runtime.Remoting.Contexts;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Rename;
+using Microsoft.VisualStudio.ProjectSystem.Waiting;
+using Microsoft.VisualStudio.Settings;
+using Solution = Microsoft.CodeAnalysis.Solution;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Rename
+{
+    internal partial class RenamerProjectTreeActionHandler
+    {
+        private readonly HashSet<Renamer.RenameDocumentActionSet> _results = new HashSet<Renamer.RenameDocumentActionSet>();
+
+        protected virtual async Task CPSCopyAsync(IProjectTreeActionHandlerContext context, IReadOnlyList<IProjectTree> items, DropEffects effect)
```

Comment:
Three-letter acronyms should have only the first letter in uppercase.

```suggestion
        protected virtual async Task CpsCopyAsync(IProjectTreeActionHandlerContext context, IReadOnlyList<IProjectTree> items, DropEffects effect)
```